src/vuex/modules/keystore.js

```diff
@@ -73,29 +73,38 @@ export default () => {
 }
 
 // creates a cosmos addres for the network desired
-function getCosmosAddressCreator(network) {
+function getCosmosAddressCreator(network, networkPrefix) {
```

Comment:
here you are duplicating code. why not default to cosmos
```suggestion
function getCosmosAddressCreator(network, networkPrefix = config.bech32Prefixes[network]) {
```

Reply:
But there you are not defaulting to cosmos right?

To default to cosmos it would be `networkPrefix = 'cosmos'`

And then the other `network` parameter becomes superfluous

Reply:
ur right, added another recommendation for slicker code